.github/ISSUE_TEMPLATE/DRAW Feature Request.yaml

```
@@ -0,0 +1,34 @@
+  - type: textarea
+    id: benefits
+    attributes:
+      label: Benefits
+      description: "Explain the benefits of the feature"
+    validations:
+      required: true
+  - type: textarea
+    id: additional_context
+    attributes:
+      label: Additional Context
+      description: "Any additional context or information"
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)